src/_includes/components/membership-item.liquid

```diff
@@ -0,0 +1,21 @@
+{% assign membershipItems = collections.membershipItems %}
+
+{% for item in membershipItems %}
+  <div class="font-sans flex flex-col items-start text-start mr-auto desktop:mb-4 desktop:mt-4 membership-item-wrapper">
+    <div class="membership-item-container" onclick="this.children[1].style.display === 'none' ? this.children[1].style.display = 'block' : this.children[1].style.display = 'none'">
+      <div class="membership-item-title" style="display: inline-flex;">
+        <div style="background: #5569B0; text-align: center; border-radius: 100%; width: 48px; height: 48px; display: block; margin-top: auto; margin-bottom: auto;">
```

Comment:
I noticed that the span element uses inline styles for styling. To align with Tailwind CSS best practices and improve maintainability, I suggest replacing the inline styles with Tailwind utility classes. Here's an updated example(double check this ahah):

`<div class="bg-[#5569B0] text-center rounded-full w-12 h-12 mx-auto my-auto block"></div>
`

https://tailwindcss.com/docs/utility-first